src/rust_api.rs

```diff
@@ -338,6 +339,16 @@ impl<T: NucleotideLike> FromStr for DnaSequence<T> {
     }
 }
 
+impl DnaSequence<Nucleotide> {
+    /// Return canonical isomorphic DNA sequence.
```

Comment:
Could also maybe use some example code here as well, again just to illustrate. I tend to prefer having the docs on the method over the iterator type, since the method is mostly what library users interact with.